src/translators/sql_translator.py

```diff
@@ -527,10 +534,9 @@ def _preprocess_values(self, e, original_attrs, col_names,
 
     @staticmethod
     def _is_ngsi_array(attr, attr_t):
-        return (attr_t == NGSI_STRUCTURED_VALUE and 'value' in attr
-                and isinstance(attr['value'], list)) \
-            or ('value' in attr and isinstance(attr['value'], list)) \
-            or attr_t == "Array"
+        return (attr_t == NGSI_STRUCTURED_VALUE
+                and 'value' in attr and isinstance(attr['value'],
+                                                   list)) or attr_t == "Array"
```

Comment:
i think you need to rebase using the proper master version

Reply:
@chicco785 , I will fork new repo and then raise the PR for same.